model/tasks/SendLtiOutcomeTask.php

```diff
@@ -118,12 +119,12 @@ private function sendLtiOutcome(taoResultServer_models_classes_OutcomeVariable $
         return true;
     }
 
-    private function buildXMLMessage($sourcedId, $grade)
+    private function buildXMLMessage($sourcedId, $grade): string
     {
         $language = 'en-us';
         $operation = 'replaceResultRequest';
         $body = '<?xml version = "1.0" encoding = "UTF-8"?>
-                <imsx_POXEnvelopeRequest xmlns = "http://www.imsglobal.org/lis/oms1p0/pox">
+                <imsx_POXEnvelopeRequest xmlns = "http://www.imsglobal.org/services/ltiv1p1/xsd/imsoms_v1p0">
```

Comment:
do we need to update a test for this ?

Reply:
not existing teset, because of usage of singleton

Reply:
I would recommend extract this XML to a factory, cover with test and call the factory here, so it is testable.